src/textures.c

```diff
@@ -306,10 +306,10 @@ typedef struct
 
 static png_texture_t pngTexture;
 
-static int texPngEnd(png_structp pngPtr, png_infop infoPtr, FILE *file, int status)
```

Comment:
for example here is `FILE *file`, not `FILE * file` @uyjulian

Reply:
It's not in a function pointer.